### PR TITLE
Remove WizardBase.ts

### PR DESCRIPTION
**Pull request by ranasaria**
Description:
Removes WizardBase.ts which is not used any more.

### CODE REVIEW

**Review by aasimkhan30 — APPROVED** [review]

File: extensions/resource-deployment/src/ui/wizardBase.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { WizardPageBase } from './wizardPageBase';
 import { WizardPageInfo } from './wizardPageInfo';
 const localize = nls.loadMessageBundle();
 
-export abstract class WizardBase<T, P extends WizardPageBase<T>, M extends Model> {
+export abstract class WizardBase<P extends WizardPageBase<WizardBase<P, M>>, M extends Model> {
```
Comment (Charles-Gagnon) [inline extensions/resource-deployment/src/ui/wizardBase.ts:15]: 1. Am I missing something or is this never even used anywhere?
2. This seems extremely convoluted - it's basically an infinite loop of generics...do you really need to have both the wizard and the pages each know about the specific types of the others?
3. What is this even fixing? I don't understand the description you left - what tests are you referring to?

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]

File: extensions/resource-deployment/src/ui/wizardBase.ts (hunk 1 above)
Comment (ranasaria) [reply extensions/resource-deployment/src/ui/wizardBase.ts:15]: 1. It is used.
2. That is the way the generics specifications that depend on each other work apparently. Luckily, there is no infinite loop of generics with this specification.
3. Added details in the description. I previously thought the title of the PR was a sufficient description.

Comment (Charles-Gagnon) [reply extensions/resource-deployment/src/ui/wizardBase.ts:15]: Where is it used? I don't see it referenced anywhere 

image

**Comment by aasimkhan30** [comment]
@ranasaria Since I got rid of all the wizards, I feel we can remove this class completely unless you want to make some additional wizard derived from it.

File: extensions/resource-deployment/src/ui/wizardBase.ts (hunk 1 above)
Comment (alanrenmsft) [reply extensions/resource-deployment/src/ui/wizardBase.ts:15]: same here, not sure why is this no longer being used and are you using this base class in your uncommitted code?

Comment (ranasaria) [reply extensions/resource-deployment/src/ui/wizardBase.ts:15]: Ok will remove. I mistakenly thought that rezourcetype wizard stl derives from it.

**Review by Charles-Gagnon — APPROVED** [review]

**Review by alanrenmsft — APPROVED** [review]

**Pull request merged**